wrapanapi/systems/openstack.py

```diff
@@ -164,6 +175,21 @@ def rename(self, new_name):
         self.raw.update(new_name)
         self.refresh()  # update raw
 
+    def auto_assign_floating_ip(self, timeout=120):
+        """Auto assign a floating IP to this VM instance.
```

Comment:
I would much rather see this as an update to `assign_floating_ip`, making `floating_ip_pool` argument optional.

Reply:
implemented